File: src/main/java/ro/ubb/dp1819/grigor/sebastian/lab1/drinks/Cappuccino.java
package ro.ubb.dp1819.grigor.sebastian.lab1.drinks;

import java.util.List;

/**
 * 2 cups water
 * 0.25 cups coffee-beans roasted
 * 100 ml foamed milk
 */
public class Cappuccino extends Drink {
    public Cappuccino(List<Ingredient> ingredientList) {
        super(ingredientList);
    }

    @Override
    public void drink() {
        System.out.println("You are drinking a cappuccino.");
    }
}

File: anaspoiala-932-dotnet/Lab01_CreationalPatterns/Entities/Cappuccino.cs
using System.Collections.Generic;

namespace CoffeeMachine.Entities
{
	public class Cappuccino : Coffee
	{
		public static new List<CoffeeIngredient> Ingredients { get; } = new List<CoffeeIngredient>
		{
			new CoffeeIngredient(2f, "cups", "water"),
			new CoffeeIngredient(0.25f, "cups", "coffee-beans", "roasted"),
			new CoffeeIngredient(100f, "ml", "milk", "foamed")
		};


		public override string Drink()
		{
			return $"Drinking a Cappuccino.";
		}
	}
}
